src/Emogrifier/CssInliner.php

```diff
@@ -626,6 +626,20 @@ private function attributeValueIsImportant($attributeValue)
         return \strtolower(\substr(\trim($attributeValue), -10)) === '!important';
     }
 
+    /**
+     * Gets the array of selectors present in the CSS provided to `inlineCss()` for which the declarations could not be
+     * applied as inline styles, but which may affect elements in the HTML.  The relevant CSS will have been placed in a
+     * `<style>` element.  The selectors may include those used within @media rules or those involving dynamic
+     * pseudo-classes (such as `:hover`) or pseudo-elements (such as `::after`).  `inlineCss()` must have been called
+     * first, otherwise an empty array is returned.
```

Comment:
I propose the method throws a `BadMethodCallException` if it is called before `inlineCss()` is called.

Reply:
I wondered about this.  Now changed and added test for the expected exception.